src/views/wallet/UserAssetsScreen.tsx

```diff
@@ -123,15 +123,9 @@ const UserAssetsScreen: React.FC = (): JSX.Element => {
   )
 
   return (
-    <>
```

Comment:
Don't remove the feature of reloading balances. I know we don't have this in our layouts, but from UX perspective an user should able to reload data (similar to `PoolsOverview`)